schema/schema-linux.json

```diff
@@ -0,0 +1,291 @@
+                             "$ref": "defs.json#/definitions/uint64Pointer"
+                         },
+                         "swap": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/resources/memory/swap",
+                             "$ref": "defs.json#/definitions/uint64Pointer"
+                         },
+                         "swappiness": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/resources/memory/swappiness",
+                             "$ref": "defs.json#/definitions/uint64Pointer"
+                         }
+                     }
+                 },
+                 "network": {
+                     "id": "https://www.opencontainers.org/jsonschema/linux/resources/network",
+                     "type": "object",
+                     "properties": {
+                         "classId": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/resources/network/classId",
+                             "type": "string"
+                         },
+                         "priorities": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/resources/network/priorities",
+                             "oneOf": [
+                               {
+                                 "type": "array",
+                                 "items": {
+                                   "oneOf": [ { "$ref": "defs-linux.json#/definitions/NetworkInterfacePriority" } ]
+                                 }
+                               },
+                               { "type": "null" }
+                             ]
+                         }
+                     }
+                 }
+             }
+         },
+         "rlimits": {
+             "id": "https://www.opencontainers.org/jsonschema/linux/rlimits",
+             "items": [
+                 {
+                     "id": "https://www.opencontainers.org/jsonschema/linux/rlimits/0",
+                     "properties": {
+                         "hard": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/rlimits/0/hard",
+                             "type": "integer"
+                         },
+                         "soft": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/rlimits/0/soft",
+                             "type": "integer"
+                         },
+                         "type": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/rlimits/0/type",
+                             "type": "integer"
+                         }
+                     },
+                     "type": "object"
+                 }
+             ],
+             "type": "array"
+         },
+         "cgroupsPath": {
+         "oneOf": [
```

Comment:
On Thu, Mar 03, 2016 at 12:41:09PM -0800, Vincent Batts wrote:
> > +         "cgroupsPath": {
> > +         "oneOf": [
> 
> Never cared to used `jq`, and looking at it now, it returns a
> compile error for `./schema.json`, where as it is a valid doc.

Can you paste your invocation?  ‘jq . schema/schema.json’ works for
me.

> Though I don't use it for formatting as it rearranges as the values.

jq doesn't mess with key ordering unless you tell it to (e.g. with
‘-S’).